ADA/ada.qmd

```diff
@@ -36,14 +35,65 @@ The ambition is to enable access to all datasets across DfE and ESFA and retire
+
+Accessing data through Databricks works slightly differently to the way that you might currently access your data in SSMS. You can read more about how Databricks works, including data storage and how to use clusters, on the [ADA guidance site](https://analytical-data-access.azurewebsites.net/get-support/how-databricks-works). 
+
+Notebooks are made up of multiple code or markdown cells. They are different to IDEs (Integrated Development Environments) like RStudio. You can read more about the differences between IDEs and notebooks and where each might be useful [on the NHS-R website](/ADA/ada.html#differences-between-ides-and-notebooks)
+
+One benefit of Databricks is the use of workflows to run your SQL code. Workflows allow you to automate the running of single or multiple SQL scripts, so if you have large amounts of data processing to do on a regular basis, you can set up a workflow to complete this overnight. Your laptop does not need to be switched on for this to work. 
+
+---
+
+# How will I know when I need to move to Databricks?
+
+At the moment, migration to Databricks is voluntary. If your team has been onboarded then you should be contacted by the ADA team with more information, and you should be offered onboarding training to learn more about the platform and how to use it. 
+
+Once your team's data has been migrated, you will **not** lose access to your existing data sources, so you will be able to dual run your code using data from SSMS and data from the Databricks SQL warehouse to ensure that everything is working as it should be. 
+
+---
+
+# What will I have to change in my existing code? Do I need to stop using RStudio and move everything to Databricks? 
+
+**There is no current expectation for you to migrate your existing R code or scripts from RStudio to Databricks notebooks.** 
+
+If you prefer using R in a notebook format and wish to use notebooks for your analysis, this is fine, but please consider the skill sets and knowledge levels in your team when doing so. For the purposes of RAP, analysis should be reproducible. If the rest of your team are unfamiliar with notebooks and will need to pick up your work at a later date, this could cause issues. The [Statistics Development Team](mailto:statistics.development@education.gov.uk) are happy to advise on this and provide guidance if you have any concerns. 
```

Comment:
I think for this section I'd put the diagrams upfront and then go into detail after? Feels like it might be easier to digest rather than going into the details of different scenarios straight off? Not everyone will care about each scenario. Subheadings might also help break up the sections a bit so it's easier for users to see what relates to them, e.g.

- initial info box / note that you don't have to do it
- diagram
- If you use SSMS
- If you use R Studio
- A note on notebooks at the end as a new option if still needed?

Reply:
Yeah I think I was trying too hard to smush things into the existing format but I like your way better

Reply:
I've left the diagram at the end because I've gone for a "if you have x then do y" description for SQL and R beforehand and I think that flows logically and provides the context upfront - if you disagree, still happy to move it around